corehq/apps/userreports/ui/widgets.py

```diff
@@ -5,4 +5,8 @@
 class JsonWidget(forms.Textarea):
 
     def render(self, name, value, attrs=None):
+        if isinstance(value, basestring):
+            # It's probably invalid JSON
```

Comment:
How confident are you about this assumption? I don't know offhand what else might trigger this condition.